internal/network/utils.go

```diff
@@ -41,3 +45,21 @@ func GetClusterAddressStack(hosts []*models.Host) (bool, bool, error) {
 	}
 	return v4, v6, nil
 }
+
+func GetConfiguredAddressFamilies(cluster *common.Cluster) (ipv4 bool, ipv6 bool, err error) {
```

Comment:
See if https://github.com/openshift/assisted-service/blob/72f7a8848f0286d873119948643cdc80c05ba2da/internal/network/utils.go#L28 can be useful. It tells which IP families a cluster has. When we implement dual-stack, it will be possible to have two machine network, two cluster networks, and two service networks - IPv4 and IPv6 each.

Reply:
It's also thoroughly unit-tested ;)

Reply:
Wanted to know the configured stacks and not the discovered ones.

Reply:
https://coreos.slack.com/archives/CDCP2LA9L/p1625754380137100?thread_ts=1625733219.115700&cid=CDCP2LA9L